.github/workflows/integration-sgx.yaml

```diff
@@ -0,0 +1,22 @@
+name: Cloud Hypervisor Tests (SGX)
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    if: github.event_name == 'pull_request'
```

Comment:
This is wrong - you've put "on: push" and then got an if check for pull_request...

Reply:
oops